keyboard/android/src/main/java/com/capacitorjs/plugins/keyboard/Keyboard.java

```diff
@@ -55,12 +56,17 @@ public Keyboard(AppCompatActivity activity) {
         this(activity, false);
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Instead of adding this to suppress all warnings, try to split the code that uses deprecated code into new functions and add the suppress warnings to that functions only, so in the future we don't miss other deprecated code that will be hidden by this annotation.